modules/dap/session.py

```diff
@@ -398,7 +398,10 @@ async def set_breakpoints_for_file(self, file: str, breakpoints: list[SourceBrea
 
 			if breakpoint.enabled:
 				enabled_breakpoints.append(breakpoint)
-				dap_breakpoints.append(breakpoint.dap)
+				bp = breakpoint.dap
+				if not bp.condition:
```

Comment:
This patch is needed as otherwise this error is raised `Error while adding breakpoints: can't parse: [json.exception.type_error.302] type must be string, but is null`

Reply:
I think the latest commit 91f7f0c  should fix this issue if you can test it out without these changes

Reply:
yep, it fixes the issue.